CHANGELOG.md

```diff
@@ -3,8 +3,9 @@
 All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
-## [2.2.0] - 2022-MM-DD
+## [2.2.0] - 2022-08-10
```

Comment:
I think this is intentionally not specified until we decide on the release date?